cogs/fun.py

```diff
@@ -326,6 +326,15 @@ async def xkcdcomic(self, ctx):
 
         await ctx.send(f"https://xkcd.com/" + str(random.randint(1,2400)) + "/")
 
+    @commands.command(aliases=['guess'])
```

Comment:
Remember to throw in a Doxygen comment above this.

Reply:
Yep, I'll take care of that. :)